### ISSUE TITLE
Update heading structure and table labelling for "List", "Month", "Week" and "Day" views to improve accessibility

### ISSUE THREAD

**Issue by jongund** [joedolson/my-calendar#33]
In the “my-calendar” view that allows you to select between “List”, “Month”, “Week” and “Day”. 
I would like to see an H2 heading used to consistently title each view. 
The heading can be used as an accessible name for the table in the “Month” and “Week” views using `aria-labelledby` attribute. 
This would make styling the page easier and give a consistent way for screen reader users to find the calendar information. 

I have added some Javascript to provide a patch on the referenced website to demonstrate the approach I would like to see. 
https://accessibleit.disability.illinois.edu/my-calendar/

**Comment by joedolson** [comment]
While you haven't done this in your scripting, I think you're wanting to remove the table's <caption> element and instead reference it with a heading that would appear above the table & relevant navigation; is that accurate? 

There are a few things I'd need to do differently, such as check what the event heading was configured as prior to setting the heading level, but it's reasonable.

It will have some back compatibility problems, like any change that alters visible HTML output, but I'm open to breaking some back compatibility in this release - there are some things that I'd like to change that I'll simply never be able to change if I don't break compatibility at some point.

**Comment by jongund** [comment]
Right there is no need for the CAPTION element if the heading is used.  I understand the pain of backwards compatibility, but I don't think the change will break anybody's usage, the style should be OK, since headings are actively styled.

Look forward to the update!

**Comment by joedolson** [comment]
Closing this issue; I committed this quite a while ago now, and I'm not sure I have any reason for leaving it open.
